vc-generator/index.js

```diff
@@ -37,15 +52,40 @@ const main = async () => {
     _incorrectCodec(validVC),
     _incorrectDigest(key),
     _incorrectCanonize(key),
-    _incorrectSigner(key)
+    _incorrectSigner(key),
+    // make sure the validVC is in the list of VCs
+    {path, data}
   ]);
   console.log('writing vcs to /credentials');
-  // write them to disk
-  await Promise.all([
-    ...vcs,
-    // add the valid vc to the list
-    {path, data}
-  ].map(writeJSON));
+  // loop through each vc and make test data for each implementation.
+  // FIXME this will become a postman collection in the future.
+  await Promise.all(vcs.flatMap(async vc => {
+      return testAPIs.map(async implementation => {
+        // get the data for the endpoint being tested
+        const endpointData = implementation[vc.data.endpoint];
+        const url = endpointData.endpoint;
+        const headers = endpointData.headers || {};
+        // expires one year for now
+        const expires = new Date(Date.now() + 365 * 24 * 60 * 60000);
+        // adds the auth header for the request here
+        vc.data.headers = await signCapabilityInvocation({
+          url,
+          method: endpointData.method || 'POST',
+          headers: {
+            ...headers,
+            date: new Date().toUTCString(),
+            expires: expires.toUTCString()
+          },
+          json: vc.data.body,
+          // FIXME set validUntil once vc refresh is up
+          expires,
+          invocationSigner,
+          capability: JSON.parse(endpointData.zcap),
+          capabilityAction: 'write'
+        });
+        return writeJSON(vc);
+      })
```

Comment:
Indentation seems off here. 
```suggestion
    return testAPIs.map(async implementation => {
      // get the data for the endpoint being tested
      const endpointData = implementation[vc.data.endpoint];
      const url = endpointData.endpoint;
      const headers = endpointData.headers || {};
      // expires one year for now
      const expires = new Date(Date.now() + 365 * 24 * 60 * 60000);
      // adds the auth header for the request here
      vc.data.headers = await signCapabilityInvocation({
        url,
        method: endpointData.method || 'POST',
        headers: {
          ...headers,
          date: new Date().toUTCString(),
          expires: expires.toUTCString()
        },
        json: vc.data.body,
        // FIXME set validUntil once vc refresh is up
        expires,
        invocationSigner,
        capability: JSON.parse(endpointData.zcap),
        capabilityAction: 'write'
      });
      return writeJSON(vc);
    })
```